File: app/src/main/java/org/fpandhis2/usescases/about/AboutComponent.java
package org.fpandhis2.usescases.about;

import org.fpandhis2.data.dagger.PerFragment;

import dagger.Subcomponent;

/**
 * QUADRAM. Created by ppajuelo on 05/07/2018.
 */
@PerFragment
@Subcomponent(modules = AboutModule.class)
public interface AboutComponent {
    void inject(AboutFragment programFragment);

}

File: app/src/main/java/org/medhis2yes/usescases/about/AboutComponent.kt
package org.medhis2yes.usescases.about

import dagger.Subcomponent
import org.medhis2yes.data.dagger.PerFragment

@PerFragment
@Subcomponent(modules = [AboutModule::class])
interface AboutComponent {
    fun inject(programFragment: AboutFragment?)
}
